### ISSUE TITLE
Iphone 5 remove iCloud with apple tech 752 v6.1 , using virtual box , the silver 6.1 cant recognise my dfu iphone5 just rotating after clicking on dfu or luad ramdisk . I tried with terminal exploit and than when i put ./ipwndfu just waiting and didn't get the iphone is in dfu mode ? Can anyone help with this ? Virtual box has recognised my iphone in dfu mode . 

### ISSUE THREAD

**Issue by OkI21** [axi0mX/ipwndfu#280]

**Comment by Ashthetik** [comment]
https://github.com/axi0mX/ipwndfu#dependencies
Check this, it's already been stated it will be unsuccessful using VMs

**Comment by OkI21** [comment]
> https://github.com/axi0mX/ipwndfu#dependencies Check this, it's already been stated it will be unsuccessful using VMs

I have installed lisbusb and is up to date

**Comment by Ashthetik** [comment]
> > https://github.com/axi0mX/ipwndfu#dependencies Check this, it's already been stated it will be unsuccessful using VMs
> 
> I have installed lisbusb and is up to date 

Yeah those dependencies for those using physical Unix/GNU systems.

Virtual Machines declare as a 100% failure
